src/clr-angular/modal/modal.ts

```diff
@@ -85,7 +85,7 @@ export class ClrModal implements OnChanges, OnDestroy {
   }
 
   open(): void {
-    if (this._open === true) {
+    if (this._open) {
```

Comment:
I thought using Typescript meant we could explicitly type properties as boolean? what's the benefit of typescript if we have to check for truthiness like it's 2009?

(you will _never_ catch me) 😈

Reply:
With `strictNullChecks` set to `true`, the both are the same. Except that *we* can't enforce anything, so we have to be flexible and make our library work for apps that are coded like it's 2009.